rust/src/domains/ffi.rs

```diff
@@ -297,8 +297,8 @@ pub extern "C" fn opendp_domains__vector_domain(
     match atom_domain.type_.contents {
         TypeContents::GENERIC { name: "AtomDomain", .. } => 
             dispatch!(monomorphize_all, [(atom_domain.carrier_type, @primitives)], (atom_domain, size)),
-        TypeContents::PLAIN("UserDomain") => monomorphize_user_domain(atom_domain, size),
-        _ => fallible!(FFI, "VectorDomain constructor only supports AtomDomain or UserDomain inner domains")
+        TypeContents::PLAIN("ExtrinsicDomain") => monomorphize_user_domain(atom_domain, size),
+        _ => fallible!(FFI, "Inner domain of VectorDomain must be AtomDomain or ExtrinsicDomain (created via user_domain)")
```

Comment:
Should the `user_name` function be renamed to `extrinsic_domain` to match? Things were in sync, but at least the names seem a little more confusing now.